spec/sidekiq/evss/disability_compensation_form/submit_form526_spec.rb

```diff
@@ -30,5 +30,22 @@
         expect { subject.new.perform(submission.id) }.to raise_error NotImplementedError
       end
     end
+
+    context 'when all retries are exhausted' do
+      let!(:form526_submission) { create(:form526_submission) }
+      let!(:form526_job_status) { create(:form526_job_status, :non_retryable_error, form526_submission:, job_id: 1) }
+
+      it 'transitions the submission to a failure state' do
+        allow(subject).to receive(:job_exhausted)
```

Comment:
Would something like `subject(:disability_comp_submit_for_526_job) { EVSS::DisabilityCompensationForm::SubmitForm526.new.job_exhausted }` work here?

Reply:
I'm not 100% clear on what you mean?  Are you proposing to call the method manually without using the `within_sidekiq_retries_exhausted_block` helper?  If so (ie. `EVSS::DisabilityCompensationForm::SubmitForm526.sidekiq_retries_exhausted` this will not actually execute the block.  We might be able to finagle something like that, but imho if it's that important to test without stubbing this, I think the "right way" would be to figure out the RSpec configuration required to include module logic in the test scope.

Can you elaborate a bit?

Reply:
I'm just spitballing ideas to get the Linting to pass

Reply:
Was taking a look at the documentation for rubocop here: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/SubjectStub

Reply:
Gottcha, thanks for following up.  So I get why it's failing, the code is in clear violation of the policy.  I'm not saying it's a bad failure or that it doesn't make sense; I'm proposing ignoring what is generally a good rule in this case because of the aforementioned reasons. : )

But, if that feels wrong, I'll just dig into the RSpec config.  It's not that I don't know how to get the test to pass, it's that I don't know if it's worth the amount of work given the context. How does that sit?  Honestly I'm happy to fix it, i just thought this seemed like a fair case for  and exception, just need to know what the platform team prefers.

Reply:
I'll just fix it, no sweat.  Thanks for the feedback.

Reply:
@LindseySaari all fixed up.  Rebased for good measure, thanks again : )